gluster_exporter/main.go

```diff
@@ -32,17 +31,79 @@ func registerMetric(name string, fn func(), intervalSeconds int64) {
 	glusterMetrics = append(glusterMetrics, glusterMetric{name: name, fn: fn, intervalSeconds: time.Duration(intervalSeconds)})
 }
 
-func getPeerID() string {
-	return *peerid
+func getWorkingDir( glusterMgmt *string ) string {
```

Comment:
Rename to `getGlusterdWorkdir`

Reply:
Argument can be removed since that is globally available

Reply:
The scope of the variable glusterMgmt is only limited to main()..same for gluster_workdir(renamed to glusterdWorkdir as per your suggestion) as well..so I can't remove it right?

Reply:
Since the number of arguments increased, please define all arguments as struct and initialize globally (Similar to https://github.com/gluster/glusterd2/blob/master/glustercli/cmd/root.go)

Reply:
I have rewritten the code in my latest commit...removed the arguments and globally declared the flags